workflows/diagnostics/tests/prognostic/_regtest_outputs/test_load_diag_data.test_Simulations[simulation0].out

```diff
@@ -43,191 +43,31 @@ variables:
 // global attributes:
```

Comment:
This regression changed order because I switched to merging all three datasets at once. rather than merging the "physics"  and "dycore" files separately.